pkg/database/db.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by sqlc. DO NOT EDIT.
 // versions:
-//   sqlc v1.28.0
+//   sqlc v1.30.0
```

Comment:
Did you generate this or was it the agent? My version should be pinned AFAIK so I'm not sure why it upgraded